tools/create_viewer_cb.py

```diff
@@ -246,14 +247,15 @@ def create_db():
   documentId INTEGER NOT NULL PRIMARY KEY,
   main TEXT NOT NULL,
   additional TEXT NULL,
-  CONSTRAINT fkDocument FOREIGN KEY (documentId) REFERENCES Document(documentId)
+  CONSTRAINT fkDocument FOREIGN KEY (documentId) REFERENCES Document(documentId) ON DELETE CASCADE
 );
 
 CREATE VIRTUAL TABLE DocumentFts USING FTS4(
   tokenize=unicode61,
   content=`DocumentText`,
   main,
-  additional
+  additional,
+  CONSTRAINT fkDocument FOREIGN KEY (rowid) REFERENCES Document(documentId) ON DELETE CASCADE
```

Comment:
Having a constraint here is not a good idea, specially with a cascade delete on a FTS table with external content
(in fact I'm surprised it's even possible)

Reply:
Actually it's possible in Sqlite, but not with Room. I forgot to remove it here.

Reply:
Fixed